File: Python Code/3_Problem26.py
# Remove Duplicates from Sorted Array

from typing import List

class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        n = len(nums)
        index = 0
        non_duplicated = 1

        for i in range(1,n):
            if nums[index]!=nums[i]:
                index += 1
                nums[index]=nums[i]
                non_duplicated += 1

        return non_duplicated

File: C++ Code/3_Problem26.cpp
// 26. Remove Duplicates from Sorted Array
// https://leetcode.com/problems/remove-duplicates-from-sorted-array/submissions/1539697350/?envType=study-plan-v2&envId=top-interview-150

class Solution {
public:
    int removeDuplicates(vector<int>& nums){
        // https://en.cppreference.com/w/cpp/algorithm/unique
        nums.erase(unique(nums.begin(), nums.end()), nums.end());

        return nums.size();
    }
};